src/libstd/collections/hash/map.rs

```diff
@@ -2445,13 +2444,25 @@ impl RandomState {
         // iteration order allows a form of DOS attack. To counter that we
         // increment one of the seeds on every RandomState creation, giving
         // every corresponding HashMap a different iteration order.
-        thread_local!(static KEYS: Cell<(u64, u64)> = {
-            Cell::new(sys::hashmap_random_keys())
+        thread_local!(static KEYS: Cell<[u64; 2]> = {
+            let mut buf = [0u8; 16];
+            // Use a constant seed on wasm32-unknown-unknown.
+            // `cfg(target = "..")` does not work right now, see:
+            // https://github.com/rust-lang/rust/issues/63217
```

Comment:
The second part of this comment can be removed, it's fine to just say that on wasm32-unknown-unknown we skip this.